modin/experimental/xgboost/xgboost_ray.py

```diff
@@ -268,10 +268,17 @@ def create_actors(num_actors):
         List of pairs (ip, actor).
     """
     num_cpus_per_actor = _get_cpus_per_actor(num_actors)
+    # starting from ray 2.6 there is a new field: 'node:__internal_head__'
+    # example:
+    # >>> ray.cluster_resources()
+    # {'object_store_memory': 1036438732.0, 'memory': 2072877467.0, 'node:127.0.0.1': 1.0, 'CPU': 8.0, 'node:__internal_head__': 1.0}
```

Comment:
@rkooo567, I wonder if the key `'node:__internal_head__'` is kind of internal stuff? Why has Ray started exposing this?

Reply:
Yeah this does seem internal. I assume it is to allow to schedule on a head node. https://github.com/ray-project/ray/commit/0ade315bec8265274c1d2c933d99d3b335c46620

Reply:
@rkooo567, should this be hidden from the user as it seems internal?

Reply:
I think cluster_resources is the developerAPI, and we don't hide resources to end users from this API. I believe it is hidden from ray status

Reply:
Got it, thanks!